dspace-server-webapp/src/test/java/org/dspace/app/rest/signposting/controller/LinksetRestControllerIT.java

```diff
@@ -674,4 +798,98 @@ public void findTypedLinkForUnDiscoverableItem() throws Exception {
         choiceAuthorityService.clearCache();
     }
 
+    @Test
+    public void getDescribedBy() throws Exception {
+        context.turnOffAuthorisationSystem();
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        String currentDateInFormat = dateFormat.format(new Date());
+        String title = "Item Test";
+        Item item = ItemBuilder.createItem(context, collection)
+                .withTitle(title)
+                .withMetadata("dc", "identifier", "doi", doi)
+                .build();
+        String responseMimeType = "application/vnd.datacite.datacite+xml";
+        context.restoreAuthSystemState();
+
+        String titleXml = "<title xmlns=\"http://datacite.org/schema/kernel-3\">" + title + "</title>";
+
+        getClient().perform(get("/signposting/describedby/" + item.getID()))
+                .andExpect(status().isOk())
+                .andExpect(content().string(Matchers.containsString(titleXml)))
```

Comment:
@NikitaKr1vonosov  please check for just the tile without extra tag otherwise if we update the crosswalk to work with datacite version 4 this test will be broken without real value

Reply:
fixed